plugins/CoreHome/vue/src/AjaxHelper/AjaxHelper.ts

```diff
@@ -174,6 +175,8 @@ export default class AjaxHelper<T = any> { // eslint-disable-line
 
   abortController: AbortController|null = null;
 
+  abortable: boolean = true;
```

Comment:
```suggestion
  abortable = true;
```

As it otherwise [won't compile](https://github.com/matomo-org/matomo/actions/runs/13061711817/job/36445971254?pr=22990#step:13:16):

```
/home/runner/work/matomo/matomo/plugins/CoreHome/vue/src/AjaxHelper/AjaxHelper.ts
  178:3  error  Type boolean trivially inferred from a boolean literal, remove type annotation  @typescript-eslint/no-inferrable-types

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```